test/e2e/vsphere_test.go

```diff
@@ -2519,6 +2519,11 @@ func TestVSphereKubernetes129UbuntuAuthenticatedRegistryMirrorCuratedPackagesSim
 		framework.WithPackageConfig(t, packageBundleURI(v1alpha1.Kube129),
 			EksaPackageControllerHelmChartName, EksaPackageControllerHelmURI,
 			EksaPackageControllerHelmVersion, EksaPackageControllerHelmValues, nil),
+		framework.WithRegistryMirrorOciNamespaces(constants.VSphereProviderName,
+			v1alpha1.OCINamespace{
+				Registry:  "857151390494.dkr.ecr.us-west-2.amazonaws.com",
```

Comment:
I believe we no longer use the build account for packages. We should update it to regional beta account private ECR.
```suggestion
				Registry:  "067575901363.dkr.ecr.us-west-2.amazonaws.com",
```